files/web/templates_page/administrado/rekursoj.html

```diff
@@ -0,0 +1,145 @@
+                    <i class="material-icons">build</i>
+                    Administrado de eksteraj rekursoj
+                </a>
+            </li>
+        </ol>
+    </div>
+    <div class="row clearfix">
+        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+            {{#if permissions.resource.create}}
+            <div class="card">
+                <div class="header">
+                    <h2>Krei novan rekurson</h2>
+                </div>
+                <div class="body">
+                    <form id="create-resource-form" autocomplete="off">
+                        <div class="form-group form-float">
+                            <div class="form-line">
+                                <input type="text" id="create-resource-form-name" name="name" class="form-control" required>
+                                <label class="form-label" for="create-resource-form-name">Nomo</label>
+                            </div>
+                        </div>
+                        <div class="form-group form-float">
+                            <div class="form-line">
+                                <input type="text" id="create-resource-form-description" name="description" class="form-control" required>
+                                <label class="form-label" for="create-resource-form-description">Priskribo</label>
+                            </div>
+                        </div>
+                        <div class="form-group form-float">
+                            <div class="form-line">
+                                <input type="url" id="create-resource-form-url" name="url" class="form-control" required>
+                                <label class="form-label" for="create-resource-form-url">Ligilo</label>
+                            </div>
+                        </div>
+                        <button type="submit" id="create-resource-form-button" class="btn btn-primary m-t-15 waves-effect">
+                            <i class="material-icons">add_box</i>
+                            <span>Krei</span>
+                        </button>
+                    </form>
+                </div>
+            </div>
+            {{/if}}
+            <div class="card">
+                <div class="header">
+                    <h2>Eksteraj rekursoj en Centra Reto</h2>
+                    <ul class="header-dropdown">
+                        <li>
+                            <i class="material-icons clickable" id="resources-table-reload">loop</i>
+                        </li>
+                    </ul>
+                </div>
+                <div class="body">
+                    <div class="table-responsive datatable">
+                        <table class="table table-bordered table-striped table-hover" id="resources-table">
+                            <thead>
+                                <tr>
+                                    <th data-name="id"              data-searchable="true">#</th>
```

Comment:
Mi pensas ke la id-numero ne nepre estas bezonata. Tio donas impreson ke iuj ligiloj pli gravas ol aliaj. Indus simple ordigi laŭ nomo kaj ne montri la id-numeron.

Reply:
Poste la ŝanĝoj, publika paĝo kaj agordpaĝo montras resursojn oridigitajn laŭ nomordo. La publika paĝo ne montras id-numeron sed la agorpaĝo montras ĝin ĉar, laŭ la nuna maniero, estas bezonata por fari ĝisdatiĝojn (t.e. la ID-numeron estas prenita de la tabelo).

Reply:
Bone, tio enordas. Tamen eblas ne montri la id-numeron sed tamen konservi ĝin.